java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -268,22 +268,18 @@ class ReferenceModeStore private constructor(
                 )
                 when (newModelsResult) {
                     is Result.Ok -> {
-                        val allBackingUpdatesSuccessful =
```

Comment:
Here's one place where we actually used the return value. But all we did was throw an exception, do we actually need that?